script/tool/lib/src/format_command.dart

```diff
@@ -206,7 +206,21 @@ class FormatCommand extends PluginCommand {
 
     final String fromPath = relativeTo.path;
 
+    const String handFormattedPragma = '// This file is hand-formatted.';
```

Comment:
I think it's worth adding a short explanatory comment here about why this exists to save someone finding the PR to understand the reason.